lib/Common/Memory/Recycler.h

```diff
@@ -152,25 +150,6 @@ struct InfoBitsWrapper{};
 #define RecyclerNewWithBarrierFinalizedClientTracked(recycler,T,...) static_cast<T *>(static_cast<FinalizableObject *>(AllocatorNewBase(Recycler, recycler, AllocFinalizedClientTrackedWithBarrierInlined, T, __VA_ARGS__)))
 #endif
 
-
-#if defined(RECYCLER_WRITE_BARRIER_ALLOC) && GLOBAL_ENABLE_WRITE_BARRIER
-#define RecyclerNew                         RecyclerNewWithBarrier
-#define RecyclerNewPlus                     RecyclerNewWithBarrierPlus
-#define RecyclerNewPlusZ                    RecyclerNewWithBarrierPlusZ
-#define RecyclerNewZ                        RecyclerNewWithBarrierZ
-#define RecyclerNewStruct                   RecyclerNewWithBarrierStruct
-#define RecyclerNewStructZ                  RecyclerNewWithBarrierStructZ
-#define RecyclerNewStructPlus               RecyclerNewWithBarrierStructPlus
-#define RecyclerNewArray                    RecyclerNewWithBarrierArray
-#define RecyclerNewArrayZ                   RecyclerNewWithBarrierArrayZ
-#define RecyclerNewFinalized                RecyclerNewWithBarrierFinalized
-#define RecyclerNewFinalizedPlus            RecyclerNewWithBarrierFinalizedPlus
-#define RecyclerNewTracked                  RecyclerNewWithBarrierTracked
-#define RecyclerNewEnumClass                RecyclerNewWithBarrierEnumClass
-#define RecyclerNewWithInfoBits             RecyclerNewWithBarrierWithInfoBits
-#define RecyclerNewFinalizedClientTracked   RecyclerNewWithBarrierFinalizedClientTracked
-#endif
```

Comment:
our clang plugin to check recycler pointer usage

---
In reply to: [90073284](https://github.com/Microsoft/ChakraCore/pull/2105#discussion_r90073284) [](ancestors = 90073284,89916552,89911621,89900815)